src/Common/test-runtime/project.json

```diff
@@ -1,61 +1,17 @@
+      "Microsoft.NETCore.Platforms": "1.0.1-beta-23506",
+      "Microsoft.NETCore.TestHost": "1.0.0-beta-23506",
+      "Microsoft.NETCore.Console": "1.0.0-beta-23506",
+      "System.IO.Compression": "4.1.0-beta-23506",
 
       "coveralls.io": "1.4",
       "OpenCover": "4.6.166",
-      "ReportGenerator": "2.3.1",
+      "ReportGenerator": "2.3.4",
 
-      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0022",
-      "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0022",
+      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0025",
+      "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0025",
 
-      "System.Collections": "4.0.11-beta-*",
-      "System.Collections.Concurrent": "4.0.11-beta-*",
-      "System.Collections.NonGeneric": "4.0.1-beta-*",
-      "System.Collections.Specialized": "4.0.1-beta-*",
-      "System.ComponentModel": "4.0.1-beta-*",
-      "System.ComponentModel.EventBasedAsync": "4.0.11-beta-*",
-      "System.Diagnostics.Contracts": "4.0.1-beta-*",
-      "System.Diagnostics.Debug": "4.0.11-beta-*",
-      "System.Diagnostics.Tools": "4.0.1-beta-*",
-      "System.Diagnostics.Tracing": "4.0.21-beta-*",
-      "System.Globalization": "4.0.11-beta-*",
-      "System.IO": "4.0.11-beta-*",
-      "System.IO.Compression": "4.1.0-beta-*",
-      "System.Linq": "4.0.1-beta-*",
-      "System.Linq.Expressions": "4.0.11-beta-*",
-      "System.Linq.Queryable": "4.0.1-beta-*",
-      "System.Net.Http": "4.0.1-beta-*",
-      "System.Net.Primitives": "4.0.11-beta-*",
-      "System.Net.WebHeaderCollection": "4.0.1-beta-*",
-      "System.Net.WebSockets": "4.0.0-beta-*",
-      "System.Net.WebSockets.Client": "4.0.0-beta-*",
-      "System.ObjectModel": "4.0.11-beta-*",
-      "System.Reflection": "4.0.11",
-      "System.Reflection.DispatchProxy": "4.0.1-beta-*",
-      "System.Reflection.Extensions": "4.0.1-beta-*",
-      "System.Reflection.Primitives": "4.0.1-beta-*",
-      "System.Reflection.TypeExtensions": "4.0.1-beta-*",
-      "System.Resources.ResourceManager": "4.0.1-beta-*",
-      "System.Runtime": "4.0.21-beta-*",
-      "System.Runtime.Extensions": "4.0.11-beta-*",
-      "System.Runtime.Handles": "4.0.1-beta-*",
-      "System.Runtime.InteropServices": "4.0.21-beta-*",
-      "System.Runtime.Serialization.Xml": "4.1.0-beta-*",
-      "System.Security.Claims": "4.0.1-beta-*",
-      "System.Security.Cryptography.X509Certificates": "4.0.0-beta-*",
-      "System.Security.Principal": "4.0.1-beta-*",
-      "System.Text.Encoding": "4.0.11-beta-*",
-      "System.Threading": "4.0.11-beta-*",
-      "System.Threading.Tasks": "4.0.11-beta-*",
-      "System.Threading.Timer": "4.0.1-beta-*",
-      "System.Xml.ReaderWriter": "4.0.11-beta-*",
-      "System.Xml.XDocument": "4.0.11-beta-*",
-      "System.Xml.XmlDocument": "4.0.1-beta-*",
-      "System.Xml.XmlSerializer": "4.0.11-beta-*",
-    
```

Comment:
I actually took CoreFx's version verbatim.  But before I add these back, let me ask "which of these need to be stated explicitly, and why?"  Obviously we want to use the correct dependencies for WCF, but we also don't want to spuriously duplicate dependencies coming from the meta packages (maintenance headache).  I can analyze them one by one, but ideally we should have the smallest possible set of differences.  Do you know offhand which they are?